src/scripts.js

```diff
@@ -22,35 +22,34 @@ const hydrationToday = document.querySelector('.hydration-today')
 const hydrationWeekly = document.querySelector('.hydration-weekly')
 
 // event listeners
-window.addEventListener('load', displayRandomUser())
-
-// functions 
+window.addEventListener('load', () => {
 Promise.all(apiCalls)
-  .then(function(apiCallsArray) {
+  .then((apiCallsArray) => {
     const usersData = apiCallsArray[0].users
-    console.log(usersData)
     // const sleepData = apiCallsArray[1].sleep
-    const hydrationData = apiCallsArray[2].hydration
+    // const hydrationData = apiCallsArray[2].hydration
     // const activityData = apiCallsArray[3].activity
-    getRandomIndex(usersData)
-    displayRandomUser(usersData)
+  getRandomIndex(usersData)
+  displayRandomUser(usersData)
   })
+  .catch(error => console.log(error))
```

Comment:
Do we need to catch within the promise.all? I'm honestly unsure! I can do some research, and we can always get rid of it later if it's unnecessary.